srcpkgs/slurm-wlm/template

```diff
@@ -19,6 +19,7 @@ distfiles="https://github.com/SchedMD/slurm/archive/slurm-${_distver}.tar.gz"
 checksum=e53e67bd0bb4c37a9c481998764a746467a96bc41d6527569080514f36452c07
 
 CFLAGS="-fcommon"
+LDFLAGS="-Wl,-z,lazy"
```

Comment:
`# fix plugin loading for musl`

Reply:
it's not just for musl. I'll add `# fix plugin loading`